docs/quickstart.ipynb

```diff
@@ -60,27 +76,30 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!curl -O https://raw.githubusercontent.com/tardis-sn/tardis/master/docs/tardis_example.yml"
+    "import numpy as np\n",
+    "from tardis import run_tardis\n",
+    "\n",
+    "_ = np.seterr(divide=\"ignore\", invalid=\"ignore\")"
    ]
   },
   {
-   "cell_type": "markdown",
-   "metadata": {},
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
    "source": [
-    "#### Running the simulation (long output) ####"
+    "sim = run_tardis(\"tardis_example.yml\", virtual_packet_logging=True, log_level=\"ERROR\")"
```

Comment:
Don't put the `log_level` to be at `ERROR`. Keep it at `WARNING` as `ERROR` is dangerous & if anything breaks, cannot be known.

Reply:
Ok.